src/common/focus_peaking.h

```diff
@@ -175,105 +180,42 @@ schedule(static) collapse(2) aligned(focus_peaking, luma_ds:64) reduction(+:sigm
 
   // Prepare the focus-peaking image overlay
 #ifdef _OPENMP
-#pragma omp parallel for simd default(none) \
-dt_omp_firstprivate(focus_peaking, luma_ds, buf_height, buf_width, six_sigma, four_sigma, two_sigma, sigma) \
-schedule(static) collapse(2) aligned(focus_peaking, luma_ds:64)
+#pragma omp parallel for default(none) \
+  dt_omp_firstprivate(focus_peaking, luma_ds, buf_height, buf_width, six_sigma, four_sigma, two_sigma) \
+  schedule(static) collapse(2)
 #endif
-  for(size_t i = 2; i < buf_height - 2; ++i)
-    for(size_t j = 2; j < buf_width - 2; ++j)
+  for(size_t i = 0; i < buf_height; ++i)
+    for(size_t j = 0; j < buf_width; ++j)
     {
+      static const uint8_t yellow[4] = { 0/*B*/, 255/*G*/, 255/*R*/, 255/*alpha*/ };
```

Comment:
would it be better to have those static const arrays outside of the loops?

Reply:
`static const` variables are stored in the global initialized-data segment, so the only difference from varying placement is the scope of visibility.  Inside the loop, the optimizer is more likely to see that the values could be replaced by a literal on an instruction (much like a local float[4] often gets optimized away to a vector register).